src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -51,13 +52,31 @@ export default function (props: SurveyTaskListProps) {
 		}
 	};
 
+	const isSurveyBlocked = (task: SurveyTask): boolean => {
+		let blocked = false;
+
+		if (tasks && props.sequential && tasks?.length > 1) {
+			if (tasks.at(0)?.id != task.id) {
```

Comment:
This is a nitpick, but I think style-wise I wouldn't use `at` here.  Since you're always indexing `0`, it doesn't seem like there's a benefit over doing `tasks[0]?`.id` here.